tests/test_bgp_af.rb

```diff
@@ -610,6 +610,102 @@ def test_dampening
   # rubocop:enable Metrics/AbcSize,Metrics/MethodLength
 
   ##
+  ## maximum_paths
+  ##
+  def maximum_paths(asn, vrf, af)
+    /ipv4/.match(af) ? af = %w(ipv4 unicast) : af = %w(ipv6 unicast)
```

Comment:
See the comment I left on line 638. If you implement that then you can delete this line on 616.